src/arch/xtensa/include/arch/drivers/timer.h

```diff
@@ -45,7 +45,6 @@ static inline int arch_timer_register(struct timer *timer,
 	ret = arch_interrupt_register(timer->irq, timer_64_handler, timer);
 	arch_interrupt_global_enable(flags);
 
-	platform_shared_commit(timer, sizeof(*timer));
```

Comment:
ok, I understand it's a dull work to got through all of these manually, if you like, I could do it at some point, or maybe someone else. But as is this patch introduces a lot of double or dangling empty lines... And if we take this as is it'll be impossible to fix them later...

Reply:
Agreed, unless you can somehow ask Checkpatch to verify the entire codebase instead of just the patches and have a "Good First Issue" issue related to fixing this.

Reply:
sed will do this in a subsequent PR (as we have unrelated double blank lines in many places and some we want to keep from xtos etc).

Reply:
There were tons of double lines even before the first patch. Now cleaned up.

Reply:
thanks for doing this! But this won't fix cases like below where blank lines are now immediately precede or follow `}` or `{` respectively... E.g. like in  src/drivers/imx/interrupt-irqsteer.c and a few other files too. E.g. src/drivers/imx/timer.c now has this:
```
void timer_unregister(struct timer *timer, void *arg)
{
	interrupt_unregister(timer->irq, arg);

}

void timer_enable(struct timer *timer, void *arg, int core)
{
	interrupt_enable(timer->irq, arg);

}

void timer_disable(struct timer *timer, void *arg, int core)
{
	interrupt_disable(timer->irq, arg);

}
```

Reply:
Will need to clean up incrementally as this PR is time consuming to update (since it touches most files).